### DISCUSSION TITLE
Couldn't find any package by glob 'io.elementary.monitor'

### DISCUSSION THREAD

**Discussion by doobes** [stsdc/monitor#399] [Q&A] [answered]
Unable to locate package io.elementary.monitor

Where did it go?

**Comment by stsdc** [comment, marked as answer]
Please use `sudo apt install com.github.stsdc.monitor`.

We are in a process of moving to official PPA.

**Reply by wout** [reply]
I've just tried installing like this but I'm getting the same error. Should this still work?

**Reply by stsdc** [reply]
If you're on elementary 7.x it should work.

**Reply by wout** [reply]
Right, I've used it there until yesterday. Just moved to os 8.
Can I do a manual build? Or does it require some work still?

**Reply by stsdc** [reply]
You should be able to build it without a problem.

**Reply by wout** [reply]
Yes, it's working, thanks! 🙂️

**Comment by ryonakano** [comment]
Closing since we now ship Monitor with the package name `io.elementary.monitor` for OS 8.
